api/envoy/api/v2/cds.proto

```diff
@@ -622,12 +622,13 @@ message Cluster {
   // :ref:`STRICT_DNS<envoy_api_enum_value_Cluster.DiscoveryType.STRICT_DNS>`,
   // or :ref:`LOGICAL_DNS<envoy_api_enum_value_Cluster.DiscoveryType.LOGICAL_DNS>`,
   // this value is used as the cluster’s DNS refresh
-  // rate. If this setting is not specified, the value defaults to 5000ms. For
-  // cluster types other than
+  // rate. The value configured must be greater than 0ms. If this setting is not specified, the
```

Comment:
As a reader I'd be pretty confused if I say this + the `{gt {nanos: 1000000}}` bit on the field. Should we say that it needs to be greater than 1ms?

Reply:
I agree. I think a better way would be:
> at least 1 ms.

Changed that!